### ISSUE TITLE
problem when running the code example/rnn/lstm_ptb.py

### ISSUE THREAD

**Issue by eecrazy** [dmlc/mxnet#904]
when I run the example/rnn/lstm_ptb.py I got the divide by zero error.

☁  rnn [master] ⚡ python lstm_ptb.py 
OpenCV is unavailable.
Loading ./data/ptb.train.txt, size of data = 929590
Unique token: 10000
Loading ./data/ptb.valid.txt, size of data = 73761
Unique token: 10000
Vocab=10000
Training swith train.shape=(46445, 20)
Training swith val.shape=(3675, 20)
batch_size=20
seq_len=35
Epoch [28] Train: NLL=20.870, Perp=1158538262.889
Epoch [56] Train: NLL=19.906, Perp=441720552.405
/users1/zyli/nlp-tools/mxnet/example/rnn/lstm.py:184: RuntimeWarning: divide by zero encountered in log
  nll = -np.sum(np.log(seq_label_probs.asnumpy())) / len(X[0,:])
Epoch [84] Train: NLL=inf, Perp=inf

Where is the problem?

**Comment by Kublai-Jing** [comment]
I believe you are using too large learning rate.

**Comment by eecrazy** [comment]
Thanks @Kublai-Jing . It solves the problem.
